tests/identify_test.py

```diff
@@ -54,6 +54,14 @@ def test_tags_from_path_simple_file(tmpdir):
     }
 
 
+def test_tags_from_path_file_with_incomplete_shebang(tmpdir):
+    x = tmpdir.join('test')
+    x.write_text('#!   \n', encoding='UTF-8')
+    assert identify.tags_from_path(x.strpath) == {
+        'file', 'text', 'non-executable',
+    }
```

Comment:
I don't think this test exercises this change -- the shebang code isn't run unless the file is made executable

Reply:
Ah.... ok :D.

Reply:
Fixed -- thank you for that note :D!